src/cloud-api-adaptor/podvm/addons/nvidia_gpu/setup.sh

```diff
@@ -34,11 +55,11 @@ if  [[ "$PODVM_DISTRO" == "ubuntu" ]]; then
 fi
 if  [[ "$PODVM_DISTRO" == "rhel" ]]; then
     dnf config-manager --add-repo http://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo
```

Comment:
The repository is added twice. Here and at line 30, right?

Reply:
"It doesn't hurt", much. But I would still vote to remove it.

If for some reason, we decide to update one and forget about the other we might have conflicts. Also this is fetching the repository file from the internet twice.

I'm missing the reasoning for keep duplicated code. Do you mind to expand on that?

Reply:
FWIW my reasoning was that leaving only one repo configuration will result in sensitivity to the calls order, I'll change it as you're also not the first one mentioning it :)